{"name": "test_buurten_failure", "status": "failed", "statusDetails": {"message": "AssertionError: Status code not match\nassert 403 == 404\n +  where 403 = <Response [403]>.status_code", "trace": "@allure.step(\"API Call `https://acc.api.data.amsterdam.nl/v1/gebieden/buurten/0363098000001 to validate whether the no record found with `failure code 404`\")\n    def test_buurten_failure():\n    \n        \"\"\"Failure API request for the buurten\"\"\"\n    \n        gebibuurtenresp = requests.get(datasetconstantgebiedenbuurten.dataset_url + \"0363098000001\")\n>       assert gebibuurtenresp.status_code == 404, \"Status code not match\"\nE       AssertionError: Status code not match\nE       assert 403 == 404\nE        +  where 403 = <Response [403]>.status_code\n\ne2etest/test_datasetgebiedenbuurten.py:139: AssertionError"}, "description": "Failure API request for the buurten", "steps": [{"name": "API Call `https://acc.api.data.amsterdam.nl/v1/gebieden/buurten/0363098000001 to validate whether the no record found with `failure code 404`", "status": "failed", "statusDetails": {"message": "AssertionError: Status code not match\nassert 403 == 404\n +  where 403 = <Response [403]>.status_code\n", "trace": "  File \"/Users/jeswaran/projects/dso-api/venv/lib/python3.8/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/Users/jeswaran/projects/dso-api/src/e2etest/test_datasetgebiedenbuurten.py\", line 139, in test_buurten_failure\n    assert gebibuurtenresp.status_code == 404, \"Status code not match\"\n"}, "start": 1647271073884, "stop": 1647271073952}], "start": 1647271073884, "stop": 1647271073952, "uuid": "e80d02ba-dd9b-467e-bfd6-c233b6e22660", "historyId": "6d2d2afe8521e49849a7d1201a789880", "testCaseId": "76f46efd1f9a250d2bb8382d9d249708", "fullName": "e2etest.test_datasetgebiedenbuurten#test_buurten_failure", "labels": [{"name": "parentSuite", "value": "e2etest"}, {"name": "suite", "value": "test_datasetgebiedenbuurten"}, {"name": "host", "value": "Jayashrees-MBP.domain_not_set.invalid"}, {"name": "thread", "value": "85835-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "e2etest.test_datasetgebiedenbuurten"}]}